controllers/noobaa-rules.yaml

```diff
@@ -0,0 +1,81 @@
+       - expr: |
+           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+         record: job:noobaa_total_unhealthy_namespace_buckets
+       - expr: |
+           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+         record: job:noobaa_namespace_bucket_count
+       - expr: |
+           NooBaa_accounts_num{namespace="redhat-data-federation"}
+         record: noobaa_accounts_num
+       - expr: |
+           NooBaa_total_usage{namespace="redhat-data-federation"}
+         record: noobaa_total_usage
+       - expr: |
+            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+         record: noobaa_num_unhealthy_namespace_resources
+       - expr: |
+           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+         record: nooBaa_num_namespace_resources
+       - expr: |
+           NooBaa_namespace_resource_status{namespace="redhat-data-federation"}
+         record: noobaa_namespace_resource_status
+       - expr: |
+           NooBaa_namespace_bucket_status{namespace="redhat-data-federation"}
+         record: noobaa_namespace_bucket_status
+   - name: noobaa-internal.rules
+     rules:
+       - expr: |
+           NooBaa_odf_health_status{namespace="redhat-data-federation"}
+         labels:
+           system_type: NooBaa
+           system_vendor: Red Hat
+         record: noobaa_system_health_status
+   - name: bucket-state-alert.rules
+     rules:
+       - alert: NooBaaNamespaceBucketErrorState
+         annotations:
+           description: A NooBaa namespace bucket {{ $labels.bucket_name }} is in error
+             state for more than 5m
+           message: A NooBaa Namespace Bucket Is In Error State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_namespace_bucket_status{bucket_name=~".*"} == 0
+         for: 5m
+         labels:
+           severity: warning
+   - name: resource-state-alert.rules
+     rules:
+       - alert: NooBaaResourceErrorState
+         annotations:
+           description: A NooBaa resource {{ $labels.resource_name }} is in error state
+             for more than 5m
+           message: A NooBaa Resource Is In Error State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_resource_status{resource_name=~".*"} == 0
+         for: 5m
+         labels:
+           severity: warning
+       - alert: NooBaaNamespaceResourceErrorState
+         annotations:
+           description: A NooBaa namespace resource {{ $labels.namespace_resource_name
+             }} is in error state for more than 5m
+           message: A NooBaa Namespace Resource Is In Error State
+           severity_level: warning
+           storage_type: NooBaa
+         expr: |
+           NooBaa_namespace_resource_status{namespace_resource_name=~".*"} == 0
+         for: 5m
+         labels:
+           severity: warning
```

Comment:
Instead of calling it namespace resource and bucket can we change the naming as per the product?

Bucket Class = Buckey Policy

Namespace Store = Data resource